ideascube/mediacenter/tests/test_views.py

```diff
@@ -10,7 +10,8 @@
 from .factories import DocumentFactory
 from ideascube.search.models import Search
 
-pytestmark = pytest.mark.django_db
+
+pytestmark = [pytest.mark.django_db]
```

Comment:
Relic of the previous version of the patch (where the pytestmark was a list of several marks)

I'll remove this chunk.